dart/dynamics/BodyNode.h

```diff
@@ -1088,9 +1035,15 @@ class BodyNode : public Frame, public SkeletonRefCountingBase
   /// allows some performance optimizations.
   std::set<Entity*> mNonBodyNodeEntities;
 
+  /// List of EndEffectors that are attached to this BodyNode
+  std::vector<EndEffector*> mEndEffectors;
+
   /// List of markers associated
   std::vector<Marker*> mMarkers;
 
+  /// Map that retrieves the cleaners for a given Node
+  std::unordered_map<Node*, std::shared_ptr<NodeCleaner> > mNodeMap;
```

Comment:
It seems okay to use `const Node*` for key type of this map as: `std::unordered_map<const Node*, std::shared_ptr<NodeCleaner>>`. Then we can remove the const cast in `Node::isRemoved()`.